fees/beraborrow/index.ts

```diff
@@ -0,0 +1,233 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import BigNumber from "bignumber.js";
+
+
+const FEE_RECEIVER = 'Oxce7d3fd53c0510325b3cebb96298522e6c538753';
+const TRANSFER_TOPIC = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef';
+const PSMContracts = ["0x6983E589E57E244B4e42FA8293B4128d15D4AaC6", "0xB2F796FA30A8512C1D27a1853a9a1a8056b5CC25"]; // OLD - NEW Contracts
+const LSPRouter = "0x3A7ED65b35fDfaaCC9F0E881846A9F4E57181446"
+const NectGasPool = "0x088D80A806b015a3047baF3e8D0A391B3D13e0c8";
+const NECT = "0x1cE0a25D13CE4d52071aE7e02Cf1F6606F4C79d3"
+const zeroAddress = "0x0000000000000000000000000000000000000000000000000000000000000000"
+
+
+const fetchFees = async (options: FetchOptions) => {
+  const dailyFees = options.createBalances();
+
+  // Withdrawal Fee (0.1% – 0.3%) -- Most are 0
+
+    const tuples = await options.api.call({
+      abi: "function getAllCollateralsAndDenManagers() view returns (tuple(address collateral, address[] denManagers)[])",
+      target: "0xFA7908287c1f1B256831c812c7194cb95BB440e6",
+    });
+    
+    const collaterals = tuples.map(t => t.collateral.toLowerCase());      
+    const assets = await options.api.multiCall({
+      abi: "address:asset",
+      calls: collaterals.map((collateral: string) => ({ target: collateral })),
+    });
+
+    const withdrawalFees = await Promise.all(
+      collaterals.map(async (collateral) => {
+        const logs = await options.getLogs({
+          target: collateral,
+          eventAbi: "event Transfer(address indexed from, address indexed to, uint256 value)",
```

Comment:
room for doublecounting fees, if some other action also sends the same token to the fee contract?